### ISSUE TITLE
Analytics.html throws error on startup

### ISSUE THREAD

**Issue by pwasiewicz** [MaKleSoft/padlock#141]
The following error is thrown on startup of newest version of padlock (2.3.1):

```javascript
analytics.html:41 Uncaught (in promise) RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at padlock.stats.get.then (analytics.html:41)
    at <anonymous>

```
![Error](https://preview.ibb.co/dbhgOG/analitycs.jpg)

So, line `new Date(stats.lastSync).toISOString()` throws mentioned exception. In my case, date range is:

```
> stats.lastSync
undefined
```

So.. here is problem, I think. What is more, error could be introduced with latest commit b4886fc6f52f5f83f3827a8fbc0eb0a8b57c5f7f

**Comment by MaKleSoft** [comment]
Fixed in 1874e8b

**Comment by MaKleSoft** [comment]
Actually, I managed to get another typo in... Now it's definitely fixed though
